### ISSUE TITLE
Add support for dark theme

### ISSUE THREAD

**Issue by Keith94** [Drive4ik/simple-tab-groups#129]
Hi!
I noticed the current STG icon doesn't play nice with Firefox's dark theme. Can you add support for automatic color-switching based on current theme? (see: https://github.com/eoger/tabcenter-redux/pull/286 if needed)

![firefox_2018-03-23_10-00-07](https://user-images.githubusercontent.com/5490615/37843131-f94a9fd4-2ebb-11e8-9f10-6820ee483392.png)

**Comment by Drive4ik** [comment]
Hi! You can manualy change this color:
![screenshot_1](https://user-images.githubusercontent.com/7843031/37844547-dbc20ec4-2ed0-11e8-9235-88a11be5e400.png)

**Comment by Keith94** [comment]
Hmm, I don't think it's the best solution because it also affects the popup menu (if you make the icon white, it's invisible in the menu). Is there a more elegant (automatic) solution?

**Comment by Drive4ik** [comment]
I understand you, I myself wanted to do so initially, but the fact is that I also need to draw a triangle in this icon with the desired color, so the icon becomes custom, and at this point I must specify the color of the icon itself. since I do not know which theme is now active, I made a setting with a manual color indication.
If you know how detect current theme (dark/ligth), please tell me.
P.S. 
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/theme/getCurrent
not working... you can see it: 
https://github.com/mdn/webextensions-examples/tree/master/theme-integrated-sidebar

**Comment by Keith94** [comment]
@Drive4ik Maybe specifying `theme_icons` in `manifest.json` could be a solution. You can check these fixes and let me know if that helps you.
* https://github.com/piroor/treestyletab/blob/master/webextensions/manifest.json#L40-L45
* https://github.com/eoger/tabcenter-redux/pull/286

**Comment by Drive4ik** [comment]
Yes, I know about this functional, but as I wrore below "I also need to draw a triangle in this icon with the desired color, so the icon becomes custom", this solution does not fit...

**Comment by Keith94** [comment]
@Drive4ik Can you make the "custom icon" optional in the settings? We can choose automatic or custom, so that automatic one will use proper colors in dark theme, etc. but without the triangle.

**Comment by Drive4ik** [comment]
Will fix in FF 63 or may be 62

**Comment by Keith94** [comment]
Thanks for the fix!

**Comment by Drive4ik** [comment]
You can try this: https://github.com/Drive4ik/simple-tab-groups/issues/213#issuecomment-406294829 to enable this feature now
